packages/fluentui/react-bindings/src/icons/createImgIcon.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { ImgIconCreateFnParams, ImgIconProps } from './types';
+import { ImgIconContext } from './imgIconContext';
+
+const defaultUrlResolver = (
+  urlConfig: { [key: string]: any } | undefined,
+  name: string,
+  props: { [key: string]: any },
+) => {
+  const { baseUrl, refreshUrl } = urlConfig || {};
+  const { size = 16, sizeModifier, type } = props || {};
+
+  let svgUrl = `${baseUrl}/${size}${
+    sizeModifier && type === 'png' ? `_${sizeModifier}` : ''
+  }/${name}.${type}?${refreshUrl}`;
```

Comment:
```suggestion
  }/${name}.${type}?${queryParam}`;
```

`queryParam` or `search` will be more universal